### ISSUE TITLE
Duplicate IDs, Aria-labels and Labels in Example and test files.

### ISSUE THREAD

**Issue by saket2403** [cerner/terra-framework#933]
## Description
Examples and Test Files of certain components have duplicate IDs, Aria-labels and label , thus giving violations while running the axe plugin.

## Components

- Date   Picker
- Date Time Picker
- Modal Manager
- Navigation Prompt
- Application Navigation
- Slide Panel Manager
